content/fundamentals/advanced.md

```diff
@@ -0,0 +1,117 @@
+### Advanced Concepts
+
+So far we've breezed over a couple of advanced topics in TypeScript like
+decorators and how the DI system is actually working under the hood.
+If all you want is a system that works, you can probably move on to another
+section, but if you want to get a deeper understanding of how Nest works, this
+is the section for you.
+
+#### The @Injectable() Decorator
+
+Earlier in the docs we mentioned that "The `@Injectable()` decorator attaches
+metadata, which declares that `CatsService` is a class that can be managed by
+the Nest IoC container.". This is only a half truth. In reality, adding the
```

Comment:
I just notice that we didn't mention what _IoC_ stands for anywhere in this page. What about:
```suggestion
the Nest IoC (Inversion of Control) container.". This is only a half truth. In reality, adding the
```

as this page is about advanced stuff, I hope one could take this as a starting point to learning more on concepts that we won't introduce here

Reply:
@jmcdo29 :ping_pong: what do you think?

Reply:
I like it! Thanks for re-pinging me on this :)